app/views/admin/reports/_rendering_options.html.haml

```diff
@@ -27,5 +27,5 @@
   .row
     .alpha.two.columns= label_tag nil, t(:report_columns)
     .omega.fourteen.columns
-      = render MultipleCheckedSelectComponent.new(name: "fields_to_show", options: @report.available_headers, selected: @rendering_options.options[:fields_to_show])
+      = render MultipleCheckedSelectComponent.new(id: 'fields_to_show', name: "fields_to_show", options: @report.available_headers, selected: @rendering_options.options[:fields_to_show])
```

Comment:
Needing to change the code just to test it can be a code smell. And system specs should test form the perspective of the user which can't know DOM ids. It's better to click on something visible. I'll add a commit to show you an alternative.

Reply:
I'll make sure to keep that in mind. Thanks :)